src/sandbox/start.js

```diff
@@ -91,6 +99,17 @@ module.exports = function _start (params, callback) {
       events.start(params, callback)
     },
 
+    // Run scheduled events from arc file (@scheduled)
+    // requires -s / --scheduled option
+    function _scheduled (callback) {
+      if (!runScheduled) {
+        let skipMessage = chalk.grey.dim('use --scheduled option to run scheduled events')
+        console.log(`- ${skipMessage}\n`)
```

Comment:
Please always use the updater, it's how we control and unify all logging.

Reply:
thanks i will fix that